api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -820,6 +820,13 @@ message HttpConnectionManager {
   // This should be set to `false` in cases where Envoy's view of the downstream address may not correspond to the
   // actual client address, for example, if there's another proxy in front of the Envoy.
   google.protobuf.BoolValue add_proxy_protocol_connection_state = 53;
+
+  // Configurations for RBAC-based authorization audit loggers.
+  // Any RBAC filter in the HTTP filter chain will invoke all the audit loggers configured here if the audit
+  // condition specified in the RBAC is met.
+  //
+  // [#not-implemented-hide:]
+  repeated config.core.v3.TypedExtensionConfig audit_log = 54;
```

Comment:
Should this be https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/accesslog/v3/accesslog.proto#config-accesslog-v3-accesslog?

Reply:
Sorry, I messed up the commit chain of this branch so decided to have a new PR #26001 for the same thing. No the access log is already present in HCM. This new field is for something different as I described above.